streaming/java/streaming-runtime/src/main/java/io/ray/streaming/runtime/master/scheduler/JobScheduler.java

```diff
@@ -0,0 +1,16 @@
+package io.ray.streaming.runtime.master.scheduler;
+
+import io.ray.streaming.runtime.core.graph.executiongraph.ExecutionGraph;
+
+/**
+ * Job scheduler is used to do the schedule job in JobMaster.
```

Comment:
```suggestion
 * Job scheduler is used to do the scheduling job in JobMaster.
```

Reply:
Job scheduler is used to do the scheduling in JobMaster.